docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/index.md

```diff
@@ -0,0 +1,60 @@
+
+## Tracking Scenarios Information
+
+- The events you want to capture on your product
+- The channels or apps where they are created
+- The actions the events represent
+- The data structure against which this event will be validated
+- The instructions that need to be applied to correctly populate the event's properties
+- The entities that should be captured together with this event
+
+This additional information represents the tracking requirements that are often lost in translation or difficult to keep up to date. With Tracking Scenarios, teams can create a shared understanding of your business' tracking needs.
+
+## Tracking Scenario Structure
+
+A Tracking Scenario consists of three main parts.
+
+### Descriptive
+
+The descriptive part includes assigning:
+
+- a name for the event described in this scenario (e.g., "Add_to_Basket");
+- a description to help people understand what action the event is capturing;
+- a series of application tags to represent which application this event needs to be tracked on;
+- a list of Triggers to describe the conditions that need to happen for the event to be captured (e.g., "When a user selects the 'Add to Basket' button on the product page of the website" and "When a user selects the '+' icon button on the product page in the mobile app").
+
+### Event Validation
+
+The event validation part allows you to set the instructions for tracking implementation related to the associated data structure. For each data structure property, you can declare where this value should be populated with:
+
+- a single exact value that you can provide (or select from the enum list associated with the data structure) (e.g., this property should always be set to "this" value);
+- a choice between a selection of values that you can provide (or select from the enum list associated with the data structure) (e.g., this property should be set with a value among this set providing certain conditions are met, which you can describe in the comment in the Detailed Instruction text box);
+- a description of what the value for this property should be;
+- any additional comments in the Detailed Instruction section.
+
+### Entities
+
+The [Entities](/docs/understanding-your-pipeline/entities/index.md) part allows you to declare which entities should be tracked with the event. You can also define whether the entity should be mandatory or optional, or whether more than one instance should be tracked with this event.
+
+Snowplow BDP provides both a UI and an API to manage your Tracking Scenarios. For information about managing Tracking Scenarios through the API, please see [Managing tracking scenarios with API](/docs/understanding-tracking-design/managing-your-tracking-scenarios/api/index.md).
+
+:::info
+See [How to manage Tracking Scenario](/docs/understanding-tracking-design/managing-your-tracking-scenarios/index.md) for more information.
+:::
```

Comment:
https://github.com/snowplow/documentation/pull/501#issuecomment-1614646216 @elyerry I added a link to "Manage tracking scenarios"